3.sumStack.js

```diff
@@ -4,36 +4,49 @@
 // 
 // Example Stack class:
 // 
-// class Stack {
-//   constructor() {
-//     this.arr = []
-//   }
-// 
-//   isEmpty() {
-//     return this.arr.length === 0
-//   }
-// 
-//   push(value) {
-//     this.arr.push(value)
-//   }
-// 
-//   pop() {
-//     return this.arr.pop()
-//   }
-// 
-//   peek() {
-//     return this.arr[this.arr.length - 1]
-//   }
-// }
+class Stack {
+  constructor() {
+    this.arr = []
+  }
+
+  isEmpty() {
+    return this.arr.length === 0
+  }
+
+  push(value) {
+    this.arr.push(value)
+  }
+
+  pop() {
+    return this.arr.pop()
+  }
+
+  peek() {
+    return this.arr[this.arr.length - 1]
+  }
+}
 // 
 // let s = new Stack()
 // stack contains: 10, 12, 267, 88, 91, 14
 //
 // sum(s) // should return 482
 
 // Implement your function here
+
+
+// queue = netflix = first in, first out
+// stack = pringles = last in, first out
+
 const sum = (stack) => {
- 
+    if(stack.isEmpty()) return 0;
+    let sum = 0;
+    whil(!stack.isEmpty()) {
+        let ele = stack.pop();
+        sum += ele;
+        stack.push(ele)
+        // return stack.push(value)
+    }
+    return sum;
 }
 
```

Comment:
you've mispelled while. Please run your code always and read the errors.

Reply:
You're also repushing into your stack with each iteration. So if the while loop was working it would run infinitely.